xbmc/guilib/GUIFontCache.h

```diff
@@ -39,6 +39,10 @@
 #include "TransformMatrix.h"
 #include "system.h"
 
+#if defined(HAS_GL) || defined(HAS_GLES)
+#include "system_gl.h"
```

Comment:
GLuint is defined as unsigned int. I don't see this this PR changes anything in behaviour.